### PR TITLE
[FIX] update multiple models due to cvxpy 1.1 (part2)

### LINKED ISSUE #2190: Reconstruction with Multi-Shell Multi-Tissue CSD
When following [the tutorial](https://dipy.org/documentation/1.1.1./examples_built/reconst_mcsd/) I get this error:

`ArpackNoConvergence: ARPACK error -1: No convergence (471 iterations, 0/1 eigenvectors converged)`

Which seems to be from the line:
`mcsd_fit = mcsd_model.fit(denoised_arr[:, :, 10:11])`

The warning preceding the error:
```
C:\fakepath\Anaconda3\envs\fakeenv\lib\site-packages\cvxpy\expressions\expression.py:516: UserWarning: 
This use of ``*`` has resulted in matrix multiplication.
Using ``*`` for matrix multiplication has been deprecated since CVXPY 1.1.
    Use ``*`` for matrix-scalar and vector-scalar multiplication.
    Use ``@`` for matrix-matrix and matrix-vector multiplication.
    Use ``multiply`` for elementwise multiplication.

  warnings.warn(__STAR_MATMUL_WARNING__, UserWarning)
```

-------

Also, from the denoising step I get this warning:

```
C:\fakepath\Anaconda3\envs\fakeenv\lib\site-packages\dipy\denoise\localpca.py:239: RuntimeWarning: invalid value encountered in true_divide
  denoised_arr = thetax / theta
```

Which I believe is from the line:
`denoised_arr = mppca(data, mask=mask, patch_radius=2)`

-------

I am using Anaconda on Windows. I have tried using Ubuntu as well and I seem to be getting the same error and warnings.

Any help you can provide is much appreciated.

### PR DESCRIPTION
**Pull request by skoudoro**
This PR fixes #2190 and more specifically the following errors/warnings:

```
/home/travis/build/dipy/dipy/venv/lib/python3.5/site-packages/cvxpy/expressions/expression.py:516: UserWarning: 
  This use of ``*`` has resulted in matrix multiplication.
  Using ``*`` for matrix multiplication has been deprecated since CVXPY 1.1.
      Use ``*`` for matrix-scalar and vector-scalar multiplication.
      Use ``@`` for matrix-matrix and matrix-vector multiplication.
      Use ``multiply`` for elementwise multiplication.
  
    warnings.warn(__STAR_MATMUL_WARNING__, UserWarning)
```

The problem was fixed in the MAPMRI module via #2188 so this PR is just an extension of it.

### CODE REVIEW

**Comment by arokem** [comment]
Thanks for doing this! Would it make sense to drop support for cvxpy < 1.1 instead? This is not a change to our public API, so it would just require users to upgrade their cvxpy installations. Granted, this is a recent version (1.1.0 was released in June 2020).

**Comment by skoudoro** [comment]
> Would it make sense to drop support for cvxpy < 1.1 instead

To me, it will make sense. However, CVXPY 1.1.0 is really too young... so we have no choice to wait a bit... 1 or 2 years. 

I do not know how often CVXPY does a release.

Let's see next summer!

**Comment by skoudoro** [comment]
Any additional comments here? this PR is ready to go!
